src/components/explorer/utils.js

```diff
@@ -136,3 +176,21 @@ export const ontologyTermSorter = (k) => (a, b) => {
 };
 
 export const explorerIndividualUrl = (individualID) => `/data/explorer/individuals/${individualID}`;
+
+export const useIndividualIgvViewableExperimentResults = (individual) => {
```

Comment:
this is only used once, so this could be put in the useEffect I commented on above